gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -227,16 +227,16 @@
+                <option value="sendreceive">{{friendlyNameFromFolderType('sendreceive').replace('&amp;amp;', '&amp;')}}</option>
+                <option value="sendonly">{{friendlyNameFromFolderType('sendonly')}}</option>
+                <option value="receiveonly">{{friendlyNameFromFolderType('receiveonly')}}</option>
+                <option value="receiveencrypted" ng-disabled="editingFolderExisting()">{{friendlyNameFromFolderType('receiveencrypted')}}</option>
               </select>
               <p ng-if="currentFolder.type == 'sendonly'" translate class="help-block">Files are protected from changes made on other devices, but changes made on this device will be sent to the rest of the cluster.</p>
               <p ng-if="currentFolder.type == 'receiveonly'" translate class="help-block">Files are synchronized from the cluster, but any changes made locally will not be sent to other devices.</p>
-              <p ng-if="currentFolder.type == 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{'Receive Encrypted' | translate}}">Stores and syncs only encrypted data. Folders on all connected devices need to be set up with the same password or be of type "{%receiveEncrypted%}" too.</p>
-              <p ng-if="editingFolderExisting() && currentFolder.type == 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{'Receive Encrypted' | translate}}">Folder type "{%receiveEncrypted%}" cannot be changed after adding the folder. You need to remove the folder, delete or decrypt the data on disk, and add the folder again.</p>
-              <p ng-if="editingFolderExisting() && currentFolder.type != 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{'Receive Encrypted' | translate}}">Folder type "{%receiveEncrypted%}" can only be set when adding a new folder.</p>
+              <p ng-if="currentFolder.type == 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{friendlyNameFromFolderType('receiveencrypted')}}">Stores and syncs only encrypted data. Folders on all connected devices need to be set up with the same password or be of type "{%receiveEncrypted%}" too.</p>
+              <p ng-if="editingFolderExisting() && currentFolder.type == 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{friendlyNameFromFolderType('receiveencrypted')}}">Folder type "{%receiveEncrypted%}" cannot be changed after adding the folder. You need to remove the folder, delete or decrypt the data on disk, and add the folder again.</p>
+              <p ng-if="editingFolderExisting() && currentFolder.type != 'receiveencrypted'" translate class="help-block" translate-value-receive-encrypted="{{friendlyNameFromFolderType('receiveencrypted')}}">Folder type "{%receiveEncrypted%}" can only be set when adding a new folder.</p>
```

Comment:
Could use `currentFolder.type` in the argument, since you've just checked that it is in fact `receiveonly`.  Less potential for future copy-paste errors.

Reply:
Removed the controller part, moving the translations back to HTML, so this is irrelevant as of now.